docs/DeveloperGuide.md

```diff
@@ -376,7 +376,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
-* **Private contact detail**: A contact detail that is not meant to be shared with others
+* **Private bug fixing progress detail**: A bug fixing progress detail that is not meant to be shared with others
```

Comment:
Should we fix the Glossary part now? Since this part defines different terms in the Guide and we don't know what in there yet.

Reply:
I think we can update later. Right now, I just followed the format of Glossary AddressBook 3 :)

Reply:
Alright I see.